src/mame/drivers/klax.cpp

```diff
@@ -88,6 +88,9 @@ void klax_bootleg_state::machine_start()
 
 	m_audio_sample[0] = m_audio_sample[1] = 0x00;
 	m_audio_nibble = 0;
+
+	save_pointer(NAME(m_audio_sample), 2);
```

Comment:
You only need `save_pointer` for indeterminate-length arrays (e.g. `std::unique_ptr<T []>`).  You don’t need it when the compiler can work out how long the array is (`T [N]` or `std::array<T, N>`).